[iOS Session Replay] Update image troubleshooting section (#30851)
DataDog/documentation: content/en/real_user_monitoring/session_replay/mobile/troubleshooting.md

Images follow a separate processing pipeline, which may introduce a small delay between uploading a replay and the image being available for rendering in the web player.
In this case, wait a few minutes and then reload the replay. 

> On iOS, vector images are not supported in Session Replay. These appear as "Unsupported image type" placeholders in the replay. To work around this, you can disable the "Preserve Vector Data" option in your asset catalog so that Xcode rasterizes the images, allowing them to be captured correctly. Note that SF Symbols are also not supported.

Reviewer: ```suggestion
On iOS, vector images are not supported in Session Replay and appear as "Unsupported image type" placeholders in the replay. To work around this, disable the "Preserve Vector Data" option in your asset catalog so that Xcode rasterizes the images, allowing them to be captured correctly. SF Symbols are also not supported.
```
